netci.groovy

```diff
@@ -117,8 +114,9 @@ def DailyBuildTaskSetup = { newJob, isPR, triggerName, groupRegex ->
     // The addition of triggers makes the job non-default in GitHub.
     if (isPR) {
         def triggerRegex = "(${dailyRegex}|${groupRegex}|${triggerName})"
-        Utilities.addGithubPRTrigger(newJob,
+        Utilities.addGithubPRTriggerForBranch(newJob,
             triggerName, // GitHub task name
+            branch,
```

Comment:
Whoops, wrong argument order here.

Also after talking with @dilijev, we may as well just backport the version from the 1.4 branch in its whole.